test/view/iota.cpp

```diff
@@ -50,6 +50,22 @@ void test_iota_minus() {
   CHECK(detail::iota_minus_(I(max), I(max)) == D(0));
 }
 
+template <typename U>
+void test_wrap_around() {
+    using namespace ranges;
+    auto max = std::numeric_limits<U>::max();
+    U from = max - 2;
+    U to = 2;
+
+    auto rng = view::ints(from, to);
+    CHECK(ranges::size(rng) == std::size_t(5));
+    ::check_equal(rng, {U(max - 2), U(max - 1), max, U(0), U(1)});
+
+    auto rng2 = view::closed_ints(from, to);
+    CHECK(ranges::size(rng2) == std::size_t(6));
+    ::check_equal(rng2, {U(max - 2), U(max - 1), max, U(0), U(1), U(2)});
+}
+
```

Comment:
While this is indeed a perversion, I think this should work for unsigned integers, and that we should assert for signed integers to prevent undefined behavior. 

I am preparing right now a PR that deprecates this and adds `view::indices/closed_indices` and for those and unsigned integers I use this behavior as well. 

Maybe fixing these for the two-argument ints version is not worth it since those who could benefit will almost immediately get a deprecated warning, but the fix is already here so... I also don't see a reason why we shouldn't apply it :/